tf/examples/Bonsai/helpermethods.py

```diff
@@ -56,7 +63,7 @@ def getArgs():
     parser.add_argument('-d', '--depth', type=checkIntNneg, default=2,
                         help='Depth of Bonsai Tree ' +
                         '(default: 2 try: [0, 1, 3])')
-    parser.add_argument('-p', '--proj-dim', type=checkIntPos, default=10,
+    parser.add_argument('-p', '--proj_dim', type=checkIntPos, default=10,
```

Comment:
Use `-`. 
That is, `--proj-dim` instead of `--proj_dim` .

When dereferencing, `-` are converted to `_` and you can use `getArgs().proj_dim`